Sources/Charcoal/Filters/Root/Views/CalloutOverlay.swift

```diff
@@ -35,7 +35,12 @@ final class CalloutOverlay: UIView {
 
     private lazy var bodyView: UIView = {
         let view = UIView(withAutoLayout: true)
-        view.backgroundColor = UIColor(white: 1, alpha: 0.8)
+        if #available(iOS 13, *),
+            traitCollection.userInterfaceStyle == .dark {
+            view.backgroundColor = UIColor.bgSecondary.withAlphaComponent(0.6)
+        } else {
+            view.backgroundColor = UIColor(white: 1, alpha: 0.8)
+        }
```

Comment:
we do import FinniversKit from charcoal.
Then you can define a private extension on UIColor to define this background and use `dynamicColorIfAvailable`

Reply:
Did that now 👍 Still needed the dark mode check, since they had different alpha components. (Thought it should be more transparent for dark mode, since the button was barely visible through the overlay).